pkg/controller/direct/alloydb/cluster_controller.go

```diff
@@ -324,17 +326,11 @@ func (a *ClusterAdapter) Create(ctx context.Context, createOp *directbase.Create
 
 	desired := a.desired.DeepCopy()
 	resource := AlloyDBClusterSpec_ToProto(mapCtx, &desired.Spec)
+	resource.Labels = common.ComputeLabels_ToProto(mapCtx, a.u)
```

Comment:
Nit: I would check the error before you assign to resource, to avoid a potential nil-pointer-exception